### DISCUSSION TITLE
2023/06/28 Data Architecture Meeting

### DISCUSSION THREAD

**Discussion by jkbhagatio** [SainsburyWellcomeCentre/aeon_mecha#209] [Meeting Agendas]
- https://github.com/SainsburyWellcomeCentre/aeon_experiments/discussions/262
- https://github.com/SainsburyWellcomeCentre/aeon_experiments/discussions/197
- https://github.com/SainsburyWellcomeCentre/aeon_experiments/discussions/281
- https://github.com/SainsburyWellcomeCentre/aeon_experiments/issues/143
- SciViz Video Streamer updates
- https://github.com/SainsburyWellcomeCentre/aeon_experiments/issues/175

**Comment by ttngu207** [comment]
Experiments (and experiment names) follows branch name for `aeon_experiments`. This is already implemented. Dario started new experiments this week using this new convention
